filter_config_client:
  ros__parameters:
    CAM_FRONT:
      translation: [1.72200568478, 0.00475453292289, 1.49491291905]
      rotation: [-0.4973392230703816, 0.49837167536166627, -0.4964832014373754, 0.5077241387638071]
    
    CAM_FRONT_RIGHT:
      translation: [1.58082565783, -0.499078711449, 1.51749368405]
      rotation: [-0.19146333228946724, 0.6785710044972951, -0.6793609166212989, 0.20335173766558642]
    
    CAM_FRONT_LEFT:
      translation: [1.5752559464, 0.500519383135, 1.50696032589]
      rotation: [-0.6687507165046241, 0.2101702448905517, -0.21108161122114324, 0.6812088525125634]
    
    CAM_BACK:
      translation: [0.05524611077, 0.0107882366898, 1.56794286957]
      rotation: [-0.4977567019405021, -0.4987849934090844, 0.496594225837321, 0.5067997344989889]
    
    CAM_BACK_RIGHT:
      translation: [1.05945173053, -0.46720294852, 1.55050857555]
      rotation: [-0.13796718183628456, -0.6893329941542625, 0.697630335509333, 0.13819187705364147]
    
    CAM_BACK_LEFT:
      translation: [1.04852047718, 0.483058131052, 1.56210154484]
      rotation: [-0.6907306801461466, -0.11209091960167808, 0.11617345743327073, 0.7048620297871717]

